### PR TITLE
(fix) Strpslashes log error

### PR DESCRIPTION
**Pull request by Lainow**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | !33520

When filling in a RichText field for the first time, an error appears on the screen.
image

This is due to the attempt to save data in the _updater_... parameter, which is an array, causing the error.
image

### CODE REVIEW

**Review by AdrienClairembault — APPROVED** [review]

File: inc/container.class.php [hunk 1]
```diff
@@ -1319,7 +1319,7 @@ public static function constructHistory(
 
             foreach ($data as $key => $value) {
                 //log only not empty values
-                if (!empty($value)) {
+                if (!empty($value) && strpos($key, '_uploader_') === false) {
```
Comment (stonebuzz) [inline inc/container.class.php:1322]:
```suggestion
                //do not log if value is empty or if dom name is part of file upload
                if (!empty($value) && strpos($key, '_uploader_') === false) {
```

**Review by stonebuzz — CHANGES_REQUESTED** [review]

**Comment by stonebuzz** [comment]
waiting for customer feedback

**Review by stonebuzz — APPROVED** [review]

**Comment by stonebuzz** [comment]
Customer approves the fixe !33520

**Review by trasher — APPROVED** [review]

**Comment by stonebuzz** [comment]
@Lainow can you complete the CHANGELOG ?

**Comment by stonebuzz** [comment]
can you rebase ?

**Pull request merged**